packages/gateway/src/services/ethereumClient.ts

```diff
@@ -47,12 +48,30 @@ export class EthereumClient<chain extends Chain> {
     } catch {
       /** error is expected when it isn't a contract */
     }
+
     return owner
   }
 
+  async getNFTOwner(node: Hex): Promise<Hex> {
```

Comment:
Express in a clearer way that this is the old implementation of ENS, before namewrapper and the other `getOwner` is supporting the NameWrapper.